docs/docs/gatsby-cli.md

```diff
@@ -1,19 +1,19 @@
 ---
-title: Commands (Gatsby CLI)
+title: Perintah (Gatsby CLI)
```

Comment:
Change title with “judul”

Reply:
wouldn't that affect when querying the frontmatter?

Reply:
Actually there is no rule for not changing that part.
But for now, let we keep it.